src/status_im/ui/components/qr_code/views.cljs

```diff
@@ -0,0 +1,63 @@
+(ns status-im.ui.components.qr-code.views
```

Comment:
Does it make sense to have `viewer` in the component name? We also have a qr code scanner component.

Reply:
Good point!